circleguard/circleguard.py

```diff
@@ -129,7 +132,7 @@ def run(self, loadables, detect, loadables2=None, max_angle=DEFAULT_ANGLE, min_d
                     # disconnect from temporary library
                     library.close()
 
-    def steal_check(self, loadables, loadables2=None):
+    def steal_check(self, loadables, loadables2=None) -> Generator[StealResult, None, None]:
```

Comment:
```suggestion
    def steal_check(self, loadables, loadables2=None) -> Iterable[StealResult]:
```
Should do the same thing but looks cleaner.

Reply:
I was surprised to see that the [python docs](https://docs.python.org/3/library/typing.html#typing.Generator) recommend typing as an `Iterable` if your generator only yields